web/ui/react-app/src/pages/targets/SeriesLabel.module.css

```diff
@@ -0,0 +1,3 @@
+.series-label {
```

Comment:
boostrap has built-in CSS classes for margins. Please refer to https://getbootstrap.com/docs/4.0/utilities/spacing/. It might be useful for such small things

Reply:
done!